### PR TITLE
fix(SplitButton storybook): - fix stories that use wrong prop

### PR DESCRIPTION
**Pull request by rivka-ungar**
https://monday.monday.com/boards/3532714909/views/113186840/pulses/6495343163

### CODE REVIEW

File: packages/core/src/components/SplitButton/__stories__/splitButton.stories.js [hunk 1]
```diff
@@ -187,7 +188,7 @@ export const SplitButtonAsThePrimaryAction = {
   render: () => (
     <SplitButton
       children="Use template"
-      secondaryPositions={SplitButton.secondaryPositions.BOTTOM_END}
+      secondaryDialogPosition={DialogPosition.BOTTOM_START}
```
Comment (YossiSaadi) [inline packages/core/src/components/SplitButton/__stories__/splitButton.stories.js:191]: You can use the secondaryDialogPosition from the SplitButton's static values. If I'm not wrong, we do not export the DialogPosition, so we want to show stories that users can just copy and use

File: packages/core/src/components/SplitButton/__stories__/splitButton.stories.js [hunk 2]
```diff
@@ -211,7 +212,7 @@ export const SecondarySplitButton = {
       <SplitButton
         children="Export to CSV"
         kind={SplitButton.kinds.SECONDARY}
-        secondaryPositions={SplitButton.secondaryPositions.BOTTOM_MIDDLE}
+        secondaryDialogPosition={DialogPosition.BOTTOM_START}
```
Comment (YossiSaadi) [inline packages/core/src/components/SplitButton/__stories__/splitButton.stories.js:215]: Same

**Review by YossiSaadi — COMMENTED** [review]
Awesome! Had a suggestion to make stories usable for users

File: packages/core/src/components/SplitButton/__stories__/splitButton.stories.js (hunk 1 above)
Comment (rivka-ungar) [reply packages/core/src/components/SplitButton/__stories__/splitButton.stories.js:191]: The issue is that the secondaryDialogPositions from the static values is of type SplitButtonSecondaryContentPosition and the secondaryDialogPosition is of type DialogPosition

Comment (rivka-ungar) [reply packages/core/src/components/SplitButton/__stories__/splitButton.stories.js]: We resolved the issue by adding SplitButtonSecondaryContentPosition type to secondaryDialogPosition, and will remove the DialogPosition type in the next major release

**Pull request merged**